TemporaryProjects.14.0/TemporaryProjects.14.0.csproj

```diff
@@ -44,6 +44,7 @@
     <WarningLevel>4</WarningLevel>
     <CreateVsixContainer>False</CreateVsixContainer>
     <DeployExtension>False</DeployExtension>
+    <StartAction>Project</StartAction>
```

Comment:
Note: by default, VS will set the first project listed in the solution file as the startup project, so that's another possible approach.

Reply:
I'm curious how you found out about this property though. Is there a setting for this somewhere in the UI? If not, you might want to consider changing this in the configuration-agnostic property group where it's still set to Program.

Reply:
Or simply remove that along with StartProgram and StartArguments. This is a class library so not-startable should be the default.

Reply:
See https://github.com/dotnet/roslyn/pull/28016 (although I'm not sure whether it's required to also move the config lines related to this project.

Reply:
Damn, I can't figure out how to stop `TemporaryProjects.14.0` from being the default start project. Nothing I try seems to work! 😭

Reply:
I'll take a look a this.

Reply:
> I'm curious how you found out about this property though. Is there a setting for this somewhere in the UI?

The project `Properties > Debug` settings were being saved to `TemporaryProjects.14.0.csproj.user`.

> I'll take a look a this.

Thanks! 🙏

Reply:
Done, I checked and it's working now.

Reply:
>  by default, VS will set the first project listed in the solution file as the startup project, so that's another possible approach.

Ah, listed in the _solution file_ not the first project displayed on solution explorer! 😆